media/js/src/exerciseSpace.tsx

```diff
@@ -129,6 +130,12 @@ export const ExerciseSpace: React.FC = () => {
     ): void => {
         evt.preventDefault();
         setShowSolutions(solutions && !showSolutions);
+        ReactGA.event({
+            category: 'Statements',
+            action: 'Clicked show solutions',
+            label: 'Clicked show solutions for question' +
```

Comment:
Verified this is working as intended. :tada:

I am wondering about tracking the level + question number (ordinal). Going forward, might the question's ordinal change as new questions are added? Is there a unique question identifier that could be used instead?

Reply:
${level},${statement.pk},${statement.premise}